apps/prairielearn/src/lib/assets.ts

```diff
@@ -179,7 +185,7 @@ export function applyMiddleware(app: express.Application) {
       immutable: !config.devMode,
     }),
   );
-  router.use('/elements/:cachebuster', elementFiles);
+  router.use('/elements/:cachebuster', coreElementFiles);
```

Comment:
Nit: `coreElementFiles` is only used here. Maybe just inline the `elementFiles` call instead of declaring it at the top of the file?

Reply:
sure, done